### ISSUE TITLE
webinterface for API needs fix

### ISSUE THREAD

**Issue by dadosch** [corona-zahlen-landkreis/corona_landkreis_fallzahlen_scraping#66]
Due to dependency updates, caused by security issues, which were closed with https://github.com/corona-zahlen-landkreis/corona_landkreis_fallzahlen_scraping/pull/61 and https://github.com/corona-zahlen-landkreis/corona_landkreis_fallzahlen_scraping/pull/24

a fix for the API webinterface is needed.

**Comment by dasmur** [comment]
Could you elaborate a little bit on this? What has changed and what, in result, fails right now?

**Comment by dadosch** [comment]
Yes, this affects only the server or api part. The crawlers are strictly in the landkreise folder. Back in March, during the Hackathon, someone tested these PRs I just merged, and noted that they break functionality. So I opened the issue in case people are wondering why it doesn't work
